src/Marten/Linq/MatchesSql/MatchesSqlExtensions.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+
+namespace Marten.Linq.MatchesSql
```

Comment:
It's following the pattern from [LastModifiedExtensions.cs](https://github.com/JasperFx/marten/blob/master/src/Marten/Linq/LastModified/LastModifiedExtensions.cs). Also, since it has to be an extension method on object, then I would want it to be opt in.